plugins/schedule/bulk_upload/batch/BulkUploadScheduleEventCsvEngine.php

```diff
@@ -0,0 +1,435 @@
+					$conflictingEvents = $this->schedulePlugin->scheduleEvent->getConflicts($bulkUploadResult->resourceId, $event);
+					if ($conflictingEvents->totalCount) {
+						//If there are conflicting events for this resource - the event should not be created.
+						KalturaLog::err('Conflicting events exist for resource ID ' . $bulkUploadResult->resourceId . ' at the specified time/s. The event will not be created.');
+						$bulkUploadResult->status = KalturaBulkUploadResultStatus::ERROR;
+						$bulkUploadResult->errorType = KalturaBatchJobErrorTypes::APP;
+						$bulkUploadResult->errorDescription .= '\n Conflicting events found for resource ID ' . $bulkUploadResult->resourceId;
+						KBatchBase::unimpersonate();
+						$bulkUploadResultChunk[] = $bulkUploadResult;
+						continue;
+					}
+				}
+				
+				$event = $this->schedulePlugin->scheduleEvent->add($event);
+				
+				// Create association between the resource and the scheduled event.
+				$scheduleEventResource = new KalturaScheduleEventResource();
+				$scheduleEventResource->resourceId = $bulkUploadResult->resourceId;
+				$scheduleEventResource->eventId = $event->id;
+				
+				
+				$this->schedulePlugin->scheduleEventResource->add($scheduleEventResource);
+				
+				$entry = $this->createTemplateEntry($bulkUploadResult);
+				
+				$updateEvent = $this->initEventObject($bulkUploadResult->eventType);
+				$updateEvent->templateEntryId = $entry->id;
+				
+				$this->schedulePlugin->scheduleEvent->update($event->id, $updateEvent);
+				KBatchBase::unimpersonate();
+				
+				$bulkUploadResult->status = KalturaBulkUploadResultStatus::OK;
+				$bulkUploadResult->templateEntryId = $entry->id;
+				$bulkUploadResultChunk[] = $bulkUploadResult;
+				
+			}
+			catch (Exception $e)
+			{
+				KalturaLog::err('An error occurred during creation of event or associated objects: ' . $e->getMessage());
+				KBatchBase::unimpersonate();
+				$bulkUploadResult->status = KalturaBulkUploadResultStatus::ERROR;
+				$bulkUploadResult->errorDescription .= '\n' . $e->getMessage();
+				$bulkUploadResult->errorType = KalturaBatchJobErrorTypes::KALTURA_API;
+				$bulkUploadResult->errorCode = $e->getCode();
+				$bulkUploadResultChunk[] = $bulkUploadResult;
+			}
+		}
+		
+		KalturaLog::info("Updating " . count($bulkUploadResultChunk) . " results");
+		
+		foreach($bulkUploadResultChunk as $result)
+		{
+			$this->addBulkUploadResult($result);
+		}
+		
+		KalturaLog::info("job[{$this->job->id}] finish creating objects");
+		
+	}
+	
+	protected function createTemplateEntry (KalturaBulkUploadResultScheduleEvent $bulkUploadResult)
+	{
+		$templateEntry = $this->initTemplateEntry($bulkUploadResult->templateEntryType);
+		$templateEntry->name = $bulkUploadResult->title;
+		$templateEntry->description = $bulkUploadResult->description;
+		$templateEntry->tags = $bulkUploadResult->tags;
+		$templateEntry->userId = $bulkUploadResult->contentOwnerId;
+		$templateEntry->creatorId = $bulkUploadResult->contentOwnerId;
```

Comment:
Was that you intention- creatorId = contentOwnerId;?

Reply:
@MosheMaorKaltura  yes, the entry's creator and user ID should be set to the contentOwnerId that comes in from the CSV.